persist/cache.go

```diff
@@ -0,0 +1,21 @@
+package persist
```

Comment:
Just a minor thing, but i think it might be convention to name the package name as `cache`. Pls correct me if I am wrong haha.

Reply:
cache pkg name is OK